.eslintrc.js

```diff
@@ -23,28 +24,20 @@ module.exports = {
   },
 
   settings: {
-    'import/core-modules': ['require', 'backburner', 'router', 'ember/version', 'node-module'],
-    'import/parsers': {
-      '@typescript-eslint/parser': ['.ts'],
-    },
+    'import/core-modules': ['require', 'backburner', 'router', '@glimmer/interfaces'],
```

Comment:
`@glimmer/interfaces` [defines modules](https://unpkg.com/@glimmer/interfaces@0.62.1/package.json) for [`main`](https://unpkg.com/@glimmer/interfaces@0.62.1/dist/commonjs/es2017/index.js), [`jsnext:main`](https://unpkg.com/@glimmer/interfaces@0.62.1/dist/modules/es5/index.js), [`module`](https://unpkg.com/@glimmer/interfaces@0.62.1/dist/modules/es2017/index.js), and [`typings`](https://unpkg.com/@glimmer/interfaces@0.62.1/dist/types/index.d.ts), but only `typings` is published to npm

Adding `@glimmer/interfaces` to `import/core-modules` still the correct types to be resolved, but `eslint-plugin-import` will ignore it..

Also note I was able to remove `ember/version`, and `node-module`